app/presenters/organisations/index_presenter.rb

```diff
@@ -40,24 +37,21 @@ def works_with(organisation)
       end
     end
 
-  private
+    def executive_office?(organisation_type)
+      organisation_type.eql?(:number_10)
+    end
+
+    def ministerial_organisation?(organisation_type)
+      executive_office?(organisation_type) || organisation_type.eql?(:ministerial_departments)
+    end
 
-    def non_ministerial_document_list(organisation_list)
-      if organisation_list
-        organisation_list.each.map do |organisation|
-          data = {
-            link: {
-              text: organisation["title"],
-              path: organisation["href"],
-              context: (I18n.t("organisations.separate_website") if organisation["separate_website"]),
-              description: works_with(organisation)
-            }
-          }
+    def link_for_non_ministerial_organisation(organisation)
+      link_title_class = "organisation-list__item-title"
 
-          data
-        end
+      if works_with_statement(organisation)
+        tag.h3(link_to(organisation["title"], organisation["href"]), class: link_title_class)
```

Comment:
This is causing some inconsistency in the output of headings:

<img width="1145" alt="screen shot 2018-06-08 at 14 10 56" src="https://user-images.githubusercontent.com/861310/41159867-dd8335da-6b25-11e8-8359-02445b33df44.png">

'Food standards Agency' is a heading but 'Crown Prosecution Service' isn't. That's a bit confusing.

Reply:
This was intentional, but perhaps this is more confusing than I intended. 

When this was in the document list component, anything with a description (i.e: "works with X public bodies") was a heading, anything else was just a link because it wasn't acting as a heading to any further content. I agree this might mean people navigating by headings miss content though/get spammed with too much content. Not sure of the best solution? Just make them all links rather than headings?

Reply:
I think so, yes. Most of them don't even count as headings, in that there's no content beneath them. Good to be consistent.